docs/user/app_getting_started.md

```diff
@@ -6,10 +6,16 @@ This document provides a step-by-step tutorial on how to get the App going and h
 
 To install the App, please follow the instructions detailed in the [Installation Guide](../admin/install.md).
 
-<!-- ## First steps with the App
+<!--
+## First steps with the App
```

Comment:
@snaselj I know this is existed previously, can we create a backlog item to address this? I would prefer this does not get inserted via drift manager.